charmcraft.yaml

```
@@ -48,18 +43,17 @@ requires:
   grafana-auth:
     interface: grafana_auth
     optional: true
-  database:
-    interface: db
+  pgsql:  # Do not call this endpoint "database" because we may want to add mysql support at some point.
+    interface: postgresql_client
     limit: 1
     optional: true
```

```suggestion
    optional: true
    description: This endpoint is specific to a PostgreSQL database backend which provides high availability for data storage. Using this endpoint overrides Grafana's usage of the embedded sqlite3 database for data storage.
```

You might have to format the string value, but you get the idea. Please double check if this statement is correct. I got it from:
- https://grafana.com/docs/grafana/latest/setup-grafana/set-up-for-high-availability/